lib/rpmrc.c

```diff
@@ -453,6 +453,44 @@ const char * lookupInDefaultTable(const char * name,
     return name;
 }
 
+#ifdef MACROFILES
+static char *initMacroPath(const char *confdir)
+{
+    return xstrdup(MACROFILES);
+}
+#else
+/*
+ * Prefer XDG_CONFIG_HOME/rpmmacros but fall back to ~/.rpmmacros
+ * if it exists and the XDG path doesn't.
+ */
+static char *initMacroPath(const char *confdir)
+{
+    const char *xdgconf = getenv("XDG_CONFIG_HOME");
+    if (!(xdgconf && *xdgconf))
+	xdgconf = "~/.config";
+    char *dotpath = rpmGetPath(xdgconf, "/rpmmacros", NULL);
```

Comment:
What about `$XDG_CONFIG_HOME/rpm/macros` instead? That mimics what we do in `/etc/rpm` too.

Reply:
+1 for the above path, too. It's also more conventional, as it seems.

Reply:
It also leaves room for future additions, so why not - will do.

Reply:
Come to think of it, not just future: there's already ~/.rpmrc that we should support in the new location too. Sigh :smile:

Reply:
Actually the ~/.rpmrc thingie did cross my mind yesterday but I just dismissed the warning light.